.readthedocs.yml

```diff
@@ -11,25 +11,26 @@ build:
   os: ubuntu-24.04
   tools:
     python: "3"
+  apt_packages:
+    - graphviz
+  jobs:
+    post_checkout:
```

Comment:
We should keep `build.commands` if possible - what was the reason for switching?

Reply:
`build.commands` is incompatible with `build.apt_packages`.  https://github.com/readthedocs/readthedocs.org/issues/9599

Reply:
Hmm, ok. Perhaps for now, could you extract the graphviz source into an e.g. `callorder.dot` file and go back to committing the generated SVG? We can then include this as either an ``.. image::`` or ``.. figure::``. Sorry for the trouble, but I now think that this the best option (and hence [`b27bcca` (#125962)](https://github.com/python/cpython/pull/125962/commits/b27bccaf8deba433d6428257d5e97de8d1faa5e6) can be reverted).

A